main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"iotdashboard/router"
+	"log"
+)
+
+func main() {
+	router, err := router.NewRouter(":8080", ":9090", "server-cert.pem", "server-key.pem")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	err = router.Ctrlr.PSQL.AddNewUser("e@g.c", "test")
+	if err != nil {
+		log.Printf("Admin user already exists in DB")
```

Comment:
Is it guaranteed that AddNewUser won't return any other error than "user already exists"? What if connection fails or something?